packages/components/layout/examples/BasicLayoutExample.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import {
+  Box,
+  Button,
+  Form,
+  FormControl,
+  TextInput,
+  Textarea,
+} from '@contentful/f36-components';
+import { Header } from '@contentful/f36-header';
+import { Layout } from '@contentful/f36-layout';
+import { NavList } from '@contentful/f36-navlist';
+
+export default function BasicLayoutExample() {
+  const [submitting, setSubmitting] = useState(false);
+  const submitForm = () => {
+    setSubmitting(true);
+    setTimeout(() => setSubmitting(false), 1000);
+  };
+
+  return (
+    <Layout
+      header={
+        <Layout.Header>
```

Comment:
We import from Header package on line 10, but still use Layout.Header here

Reply:
This is intentional 👍